### ISSUE TITLE
'&' -> '&amp;' problem

### ISSUE THREAD

**Issue by junfengP** [codemanki/cloudscraper#319]
_Please attempt to answer the following questions before submitting a new issue:_

  * What version of Cloudscraper are you using?
    - `4.5.0`
  * What version of Node.js are you using? (Please share the process information)
    - `process {
  version: 'v12.14.1',
  versions: {
    node: '12.14.1',
    v8: '7.7.299.13-node.16',
    uv: '1.33.1',
    zlib: '1.2.11',
    brotli: '1.0.7',
    ares: '1.15.0',
    modules: '72',
    nghttp2: '1.40.0',
    napi: '5',
    llhttp: '2.0.1',
    http_parser: '2.8.0',
    openssl: '1.1.1d',
    cldr: '35.1',
    icu: '64.2',
    tz: '2019c',
    unicode: '12.1'
  },
  arch: 'x64',
  platform: 'linux',
  release: {
    name: 'node',
    lts: 'Erbium',
    sourceUrl: 'https://nodejs.org/download/release/v12.14.1/node-v12.14.1.tar.gz',
    headersUrl: 'https://nodejs.org/download/release/v12.14.1/node-v12.14.1-headers.tar.gz'
  },`
  * When did the problem start occurring?
I am working on another project [RSSHub](https://github.com/DIYgod/RSSHub), I am writing some code to fetch data from `javlibrary`.  
The URL is `http://www.javlibrary.com/cn/userowned.php?list&u=siccalol`.
From the console log,  I can see `&` is escaped to `&amp;` whicl leads to `HTTP 404`.
![image](https://user-images.githubusercontent.com/15406392/73625046-35c2bd80-467e-11ea-9328-d441cb60b42a.png)

  * How often does the problem occur?
        Always
  * What is the URL?
        `http://www.javlibrary.com/cn/userowned.php?list&u=siccalol`
  * Are there any similar issues? (Please share the links)
        I have read #289 , It doesn't works for me.

- [x] I have read the [README](https://github.com/codemanki/cloudscraper#readme). (Code [examples](https://github.com/codemanki/cloudscraper/tree/master/docs/examples#readme))

_Please share a minimal working code snippet that reproduces the problem._
<details><summary>Code snippet</summary>

```js
const uid = 'siccalol';
const link = `http://www.javlibrary.com/cn/userowned.php?list&u=${uid}`;
const response = await cloudscraper.get(link); 
```
</details>

**Comment by KostyaChoporov** [comment]
I also met the same issue on another website. Cloudflare adds ``&amp;`` instead of ``&`` in the form action

**Comment by codemanki** [comment]
Should be fixed in `4.6.0`. Please update and try again
